containers/multisite/wp-config.php

```diff
@@ -93,14 +94,13 @@
 }
 
 /* Multisite */
-
-// define('WP_ALLOW_MULTISITE', true);
-define('MULTISITE', true);
+define('WP_ALLOW_MULTISITE', true);
```

Comment:
doing this breaks the current dev experience, in which running the multisite container provides a working subfolder-based network out-of-the-box. What's your motivation for this change?